python_modules/dagster-graphql/dagster_graphql/implementation/fetch_assets.py

```diff
@@ -368,9 +372,21 @@ def get_freshness_info(
         },
     )
 
+    previous_latest_record = data_time_queryer.get_most_recent_materialization_record(
+        asset_key=asset_key, before_cursor=latest_record.storage_id
+    )
+    if previous_latest_record is None:
+        return GrapheneAssetFreshnessInfo(
+            currentMinutesLate=current_minutes_late,
+            latestMaterializationMinutesLate=None,
+        )
+    previous_used_data_times = data_time_queryer.get_used_data_times_for_record(
+        record=previous_latest_record
+    )
+
     latest_materialization_minutes_late = freshness_policy.minutes_late(
         evaluation_time=latest_materialization_time,
-        used_data_times=used_data_times,
+        used_data_times=previous_used_data_times,
         available_data_times={key: latest_materialization_time for key in used_data_times.keys()},
```

Comment:
I'm not 100% sure what this parameter is providing but I wonder if this second reference to `used_data_times` should also be `prev_used_data_times`?